rest/config.go

```diff
@@ -386,8 +386,8 @@ func (dbConfig *DbConfig) setup(name string) error {
 // If the specified path does not qualify for a valid file or an URI, it returns the input path
 // as-is with the assumption that it is an inline JavaScript source. Returns error if there is
 // any failure in reading the JavaScript file or URI.
-func loadJavaScript(path string) (js string, err error) {
-	rc, err := readFromPath(path)
+func loadJavaScript(path string, insecureSkipVerify bool) (js string, err error) {
```

Comment:
What error gets logged if SG attempts to load config javascript via https that's using a self-signed cert, but the new flag isn't set?  If possible, it would be nice to add information about the new flag in the error message.

Reply:
The blow error gets logged if SG attempts to load config javascript via https that's using a self-signed cert, but the new flag isn't set:

Error loading JavaScript (SyncFunction) from "https://localhost:5000/sync", Err: Get https://localhost:5000/sync: x509: certificate signed by unknown authority. SSL certificate verification is turned on by default when loading config over HTTPS. Can be overriden by the unsupported "remote_config_tls_skip_verify" option